django/db/transaction.py

```diff
@@ -25,6 +26,16 @@ def get_connection(using=None):
     return connections[using]
 
 
+def get_aconnection(using=None):
```

Comment:
Should this be `aget_connection`? And should it not be `async def`?

Reply:
I've decided to postpone `django.db.transaction` for now, but to answer your question: it's `async def` because it doesn't need to, and I have decided not to call it `aget_connection` because it is not an `async def`, although it does return an async connection